internal/operands/node-labeller/resources.go

```diff
@@ -0,0 +1,295 @@
+package node_labeller
+
+import (
+	"kubevirt.io/ssp-operator/internal/common"
+
+	secv1 "github.com/openshift/api/security/v1"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Resources from node-labeller
+const (
+	ServiceAccountName = "kubevirt-node-labeller"
+	DaemonSetName      = "kubevirt-node-labeller"
+	ConfigMapName      = "kubevirt-cpu-plugin-configmap"
+
+	nfdVolumeName            = "nfd-source"
+	nfdVolumeMountPath       = "/etc/kubernetes/node-feature-discovery/source.d/"
+	configMapVolumeName      = "cpu-config"
+	configMapVolumeMountPath = "/config"
+)
+
+var commonLabels = map[string]string{
+	"app": "kubevirt-node-labeller",
+}
+
+var cpuPluginConfigmap = `
+    obsoleteCPUs:
+      - "486"
+      - "pentium"
+      - "pentium2"
+      - "pentium3"
+      - "pentiumpro"
+      - "coreduo"
+      - "n270"
+      - "core2duo"
+      - "Conroe"
+      - "athlon"
+      - "phenom"
+    minCPU: "Penryn"
+`
+
+var configMapData = map[string]string{
+	"cpu-plugin-configmap.yaml": cpuPluginConfigmap,
+}
+
+func ClusterRoleName(namespace string) string {
+	const clusterRoleName = "kubevirt-node-labeller"
+	return clusterRoleName + "-" + namespace
```

Comment:
We need to keep the naming scheme of all deployed resources, so we shouldn't include the resource's namespace in its name.
We have a task in progress to create a webhook to prevent creation of multiple SSP CR creations so this won't be an issue anymore

Reply:
Done